sdk/app_cpu1/common/drv/encoder.h

```diff
@@ -9,9 +9,11 @@
 void encoder_init(void);
 
 void encoder_set_pulses_per_rev_bits(uint32_t bits);
-
+void encoder_set_pulses_per_rev(uint32_t pulses);
 void encoder_get_steps(int32_t *steps);
+void encoder_get_steps_instantaneous(int32_t *steps);
 void encoder_get_position(uint32_t *position);
+void encoder_get_pos_instantaneous(uint32_t *position);
```

Comment:
Rename this to be the full `position` name (to make the other function)

Reply:
Function is now `encoder_get_position_instantaneous`